<launch>
	<param name="num_blocks" value="5" type="int"/>
	<param name="configuration" value="stacked_descending" />
        <!-- environment can be "symbolic", "simulator", "robot" -->
	<param name="environment" value="simulator" />
	<param name="limb" value="right" />
	<param name="num_arms" value="2" />

	<node name="robot_interface" pkg="zic_proj1" type="robot_interface.py" output="screen">
	</node>
	<node name="controller" pkg="zic_proj1" type="controller.py" output="screen"/>
</launch>
